services/QuillLMS/client/app/bundles/Staff/components/comprehension/semanticRules/modelsTable.tsx

```diff
@@ -25,11 +29,13 @@ const ModelsTable = ({ activityId, prompt }) => {
         const activateLink = (
           <Link className="data-link" to={`/activities/${activityId}/semantic-labels/${prompt.id}/model/${id}/activate`}>Activate</Link>
         );
+
         return {
           id: id,
           created_at: moment(created_at).format('MM/DD/YY'),
           version: older_models + 1,
           name,
+          notes: stripHtml(model.notes),
```

Comment:
If we strip the HTML here does that reduce the value of the data in this column?  I assume people want it to be viewable here in a similar manner as it is in the editor, but maybe that's too complex to solve right now?

Reply:
I used stripHtml to avoid rendering raw HTML markup, but you're right that this removes the note's styling. @emilia-friedberg Do we have a way to render HTML within DataTable?

Reply:
You can always do `<div dangerouslySetInnerHTML={{ __html: model.notes }} />` any time you want to render actual HTML. That's a general React thing, not specific to this component.